cleanlab/multiannotator.py

```diff
@@ -172,19 +172,30 @@ def get_label_quality_multiannotator(
 
     """
 
-    if isinstance(labels_multiannotator, np.ndarray):
-        labels_multiannotator = pd.DataFrame(labels_multiannotator)
+    if isinstance(labels_multiannotator, pd.DataFrame):
```

Comment:
can you confirm none of the user-facing methods' APIs change in this PR?

Reply:
Yes, user facing API has not changed. All the multiannotator functions still allow `labels_multiannotator` to be either a np array or pd dataframe. 

The difference here is that we used to convert everything to a dataframe before performing internal operations, whereas now we convert everything to a np array internally since the optimized functions operate on np arrays.